credentialMixin.js

```diff
@@ -79,31 +81,52 @@ export const credentialMixin = {
       return this.issuer.name;
     },
     credentialImage() {
+      if(!this.credential) {
+        return '';
+      }
       if(this.credential.image) {
         return this.credential.image;
       }
-      if(this.issuer.image) {
-        return this.issuer.image;
-      }
-      if(this.issuer.logo) {
-        return this.issuer.logo;
+      const {credentialSubject = {}} = this.credential;
+      if(credentialSubject.image) {
+        return credentialSubject.image;
       }
       return '';
     },
     truncateDescription() {
+      // if there is no description don't truncate it
+      if(!(this.credential && this.credential.description)) {
+        return false;
+      }
       const {length} = this.credential.description;
       return length > MAX_CHARACTER_COUNT;
+    },
+    /**
+      * Takes all of the types after the initial VC one
+      * and turns them into a string.
+    */
+    credentialType() {
+      if(!this.credential) {
+        return false;
+      }
+      const tail = this.credential.type.slice(1);
```

Comment:
I'm not sure if "VerifiableCredential" must strictly be first?  Should really have a library with helpers for this sort of thing, but might be more robust (for properly compacted JSON-LD) with:
```js
  return this.credential.type.filter(type => type !== 'VerifiableCredential').join(', ');
```

Reply:
My bad I thinking of the context:

> The value of the @context property MUST be an ordered set where the first item is a URI with the value https://www.w3.org/2018/credentials/v1. 


so yeah I will change to your filter example.

Reply:
VC type order is addressed here: https://github.com/digitalbazaar/bedrock-vue-credential-card/pull/21/commits/158dc32b4c99991ebde5290e218b666f860082aa